src/styles/blocks/monthLength.scss

```diff
@@ -0,0 +1,9 @@
+.calendar {
+  @for $number from 28 through 31 {
```

Comment:
The same according to comment above about separate logic. Write it in calendar.scss file